Parser/asdl.py

```diff
@@ -304,9 +306,9 @@ def visit(self, object, *args):
             return
         try:
             meth(object, *args)
-        except Exception, err:
-            print "Error visiting", repr(object)
-            print err
+        except Exception as err:
```

Comment:
This syntax is not compatible with Python 2.5. Use `sys.exc_info()` for more compatibility.